RecoParticleFlow/PFSimProducer/plugins/EcalBarrelClusterFastTimer.cc

```diff
@@ -172,7 +172,7 @@ float EcalBarrelClusterFastTimer::correctTimeToVertex(const float intime, const
                                                       const CaloSubdetectorGeometry* ecalGeom) const {
   if( timeDet.rawId() == 0 ) return -999.;
   // correct the cluster time from 0,0,0 to the primary vertex given
-  const CaloCellGeometry* cellGeometry ( ecalGeom->getGeometry( timeDet ) ) ;
+  auto cellGeometry = ecalGeom->getGeometry(timeDet).get();
```

Comment:
This should also not call `get()` since it can lead to a temporary `std::shared_ptr`.